cmd/tapa/main.go

```diff
@@ -83,6 +83,21 @@ func main() {
 	}
 
 	logger := log.New("Meridio-tapa", config.LogLevel)
+	logger.Info("Configuration read", "config", config)
+	if err := config.IsValid(); err != nil {
+		log.Fatal(logger, "config.IsValid", "error", err)
+	}
+
+	if config.LogLevel == "TRACE" {
+		nsmlog.EnableTracing(true) // enable tracing in NSM
+		logrus.SetLevel(logrus.TraceLevel)
```

Comment:
Do we still need logrus?

Reply:
Yes, unfortunately. NSM requires this since they are checking internally.  I had a comment on it before, which I should bring back.